src/containers/FilteredRecipesContainer.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * Contains business logic for Recipes presentation classes
+ */
+
+import { connect } from 'react-redux'
```

Comment:
Turned out to be harder than expected https://github.com/pjazdzewski1990/Learn-JS/commit/8711d187e725091c01e9a185c6e998ee976cd157